master.yaml

```diff
@@ -298,6 +321,12 @@ resources:
         - path: /usr/local/bin/retry
           permissions: 0755
           content: {get_file: fragments/retry.sh}
+        - path: /etc/sysconfig/network-scripts/ifcfg-eth1
+          content:
+            str_replace:
+              params:
+                $IFNAME: eth1
```

Comment:
I wonder if there is a reason to make this parametrized if eth1 is hardcoded on other places anyway?